src/query/functions/temporal/base.go

```diff
@@ -266,12 +262,15 @@ func parallelProcess(
 			series     = iter.Current()
 			datapoints = series.Datapoints()
 			stats      = series.Stats()
+			seriesMeta = metas[i]
 		)
 
 		if stats.Enabled {
 			decodeDuration += stats.DecodeDuration
 		}
 
+		seriesMeta.Tags = seriesMeta.Tags.WithoutName()
```

Comment:
Perhaps add `// rename series to exclude their __name__ tag as part of function processing.` above this line for consistency with other code path and explaining why without name is called?